care/facility/admin.py

```
@@ -181,19 +180,9 @@ class PatientTestGroupAdmin(admin.ModelAdmin):
-        return render_to_csv_response(
-            queryset,
-            field_header_map=FacilityUser.CSV_MAPPING,
-            field_serializer_map=FacilityUser.CSV_MAKE_PRETTY,
-        )
-
 
 @admin.register(FacilityUser)
-class FacilityUserAdmin(DjangoQLSearchMixin, admin.ModelAdmin, ExportCsvMixin):
+class FacilityUserAdmin(DjangoQLSearchMixin, admin.ModelAdmin):
     djangoql_completion_enabled_by_default = True
     actions = ["export_as_csv"]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)